ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_completion_service.proto

```diff
@@ -69,6 +69,7 @@ message CompletionStreamRequest {
   repeated string parties = 3;
 
   // This field indicates the minimum offset for completions. This can be used to resume an earlier completion stream.
+  // This offset is exclusive: the response will not contain any command whose offset is less than or equal to this.
```

Comment:
I find negation always a bit harder to read. Maybe ‘will only contain commands whose offset is strictly greater to this`?

Reply:
Ah, thanks, I struggled a bit with this!

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8145/commits/bb8621be35dd50afa01c52909147198a72e84fed